docs/examples/modules/DropdownExample/DropdownDirection.example.vue

```diff
@@ -0,0 +1,24 @@
+<template lang="html">
+  <div id="app">
+  <sui-dropdown text="File" direction="upward">
+    <sui-dropdown-menu>
+      <sui-dropdown-item>New</sui-dropdown-item>
+      <sui-dropdown-item>Open... <span class="description">ctrl + o</span></sui-dropdown-item>
+      <sui-dropdown-item>Save as... <span class="description">ctrl + s</span></sui-dropdown-item>
+      <sui-dropdown-item>Rename <span class="description">ctrl + r</span></sui-dropdown-item>
+      <sui-dropdown-item>Make a copy</sui-dropdown-item>
+      <sui-dropdown-item><sui-icon name="folder" />Move to folder</sui-dropdown-item>
+      <sui-dropdown-item><sui-icon name="trash" />Move to trash</sui-dropdown-item>
+      <sui-dropdown-divider/>
+      <sui-dropdown-item>Download as...</sui-dropdown-item>
+    </sui-dropdown-menu>
+  </sui-dropdown>
+</div>
+
+</template>
+
+<script>
+export default {
+  name: 'DropdownDirectionExample',
+};
+</script>
```

Comment:
Love this, but I think it should only change the default opening direction, while it should still go to the other side if there isn't enough space. Nobody wants the dropdown to go out of the screen.
The react implementation has a boolean `upward` prop, what do you think about that?

Reply:
I guess you're right: https://github.com/Semantic-Org/Semantic-UI/blob/master/src/definitions/modules/dropdown.js#L2407

At the moment they don't offer a way to do what you're proposing, though. I'd say let's go for `auto` and `upward`, like in the original API. I [opened an issue on SemanticUI](https://github.com/Semantic-Org/Semantic-UI/issues/6187) to see what they think about it: as soon as they point toward a possible solution we can implement it without waiting for their changes.

Reply:
> At the moment they don't offer a way to do what you're proposing, though. 

Why? according to the [docs](https://semantic-ui.com/modules/dropdown.html#/settings) they have `auto`, `downward` and `upward` 

> When set to `auto` determines direction based on whether dropdown can fit on screen. Set to `upward` or `downward` to always force a direction.

i've checked it, when initialize dropdown with `direction: downward` - menu opens downward no matter what

Reply:
Ok then, let's go for the `auto-upward` solution